code/client/cl_console.c

```diff
@@ -682,7 +695,7 @@ void Con_DrawNotify( void )
 				currentColorIndex = colorIndex;
 				re.SetColor( g_color_table[ colorIndex ] );
 			}
-			SCR_DrawSmallChar(cl_conXOffset->integer + con.xadjust + (x + 1 - (con_timestamp->integer ? 9 : 0))*smallchar_width, v, text[x] & 0xff);
+			SCR_DrawSmallChar(con_notifyx->integer + con.xadjust + (x + 1 - (con_timestamp->integer ? 9 : 0)) * SMALLCHAR_WIDTH, v + con_notifyy->integer, text[x] & 0xff);
```

Comment:
You should use `smallchar_width` instead of `SMALLCHAR_WIDTH`, otherwise when `vidHeight >> 1080` => all chars will be squashed together. Could you also change this for all other occurences in Con_DrawNotify()?